Frontend.Tests/ControllerTests/Projects/FeaturesControllerTests.cs

```diff
@@ -138,18 +164,33 @@ public async void GivenUrnAndInitiator_RedirectsProjectToFeaturesSummary()
                 [Fact]
                 public async void GivenEmptyInitiator_AddAnErrorMessageToThePageAndDoNotUpdateTheModel()
                 {
-                    var response = await _subject.InitiatedPost("0001", TransferFeatures.ProjectInitiators.Empty);
-                    var viewModel = GetViewModel(response);
+                    FeaturesInitiatedViewModel vm = new FeaturesInitiatedViewModel
+                    {
+                        Urn = "0001",
+                        WhoInitiated = TransferFeatures.ProjectInitiators.Empty
+                    };
+
+                    var results = await ControllerTestHelpers.ValidateAndAddToModelState(new FeaturesInitiatedValidator(), vm, _subject.ModelState);
+                    var response = await _subject.InitiatedPost(vm);
+
+                    //Assert
                     _projectRepository.Verify(r => r.Update(It.IsAny<Project>()), Times.Never);
-                    Assert.True(viewModel.FormErrors.HasErrors);
-                    var error = viewModel.FormErrors.Errors[0];
-                    Assert.Equal("Select who initiated the project", error.ErrorMessage);
+
+                    Assert.False(results.IsValid);
+                    Assert.Equal("WhoInitiated", results.Errors[0].PropertyName);
+                    Assert.Equal("Select who initiated the project", results.Errors[0].ErrorMessage);
+                    ControllerTestHelpers.GetViewModelFromResult<FeaturesInitiatedViewModel>(response);
```

Comment:
Not sure if this is still needed or if we need to make an assertion on the result of it

Reply:
`ControllerTestHelpers.GetViewModelFromResult<FeaturesInitiatedViewModel>(response);`
Good point, this method actually does two asserts but the name is confusing as it isn't obvious it does that. Happy to rename this method maybe

Reply:
Ah yeh I think I was looking at something else - didn't spot the 2 assertions in there! Ignore that one :)

Reply:
I have renamed to make this clearer now,